### PR TITLE
Fix a bunch of stuff in GitHub Actions test workflows

### PR DESCRIPTION
**Pull request by ewels**

### CODE REVIEW

**Comment by ewels** [comment]
Feel free to merge, despite the failing tests - the point of the PR (to run the tests) is working :sweat_smile:

File: software/fastqc/main.nf [hunk 1]
```diff
@@ -26,6 +26,7 @@ process FASTQC {
     if (single_end) {
         """
         [ ! -f  ${name}.fastq.gz ] && ln -s $reads ${name}.fastq.gz
+        ls -la
```
Comment (grst) [inline software/fastqc/main.nf:29]: Is this on purpose or left over from debugging?

**Comment by grst** [comment]
other than that, LGTM!

File: software/fastqc/main.nf (hunk 1 above)
Comment (ewels) [reply software/fastqc/main.nf:29]: Debugging 👀

**Pull request merged**